packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx

```diff
@@ -73,7 +70,15 @@ export const StripeBuyAudioButton = () => {
         amount,
         destinationWallet: (await getRootSolanaAccount()).publicKey.toString()
       })
-      const stripeOnRampInstance = StripeOnRamp(STRIPE_PUBLISHABLE_KEY)
+      if (!STRIPE_PUBLISHABLE_KEY) {
+        throw new Error('Stripe publishable key not found')
+      }
+      const stripeOnRampInstance = await loadStripeOnramp(
```

Comment:
Function was already `async` and using an `await` so I wouldn't sweat it.

Just worth noting that event handlers aren't going to block on async functions. If I understand correctly, most of this code is going to run after the click handler has had a chance to finish it's capture/bubble cycle and return back to React.